sui_programmability/examples/defi/sources/flash_lender.move

```diff
@@ -2,12 +2,12 @@
 // SPDX-License-Identifier: Apache-2.0
 
 /// A flash loan that works for any Coin type
-module DeFi::FlashLender {
-    use Sui::Balance::{Self, Balance};
-    use Sui::Coin::{Self, Coin};
-    use Sui::ID::{Self, ID, VersionedID};
-    use Sui::Transfer;
-    use Sui::TxContext::{Self, TxContext};
+module defi::flash_lender {
+    use sui::Balance::{Self, Balance};
+    use sui::Coin::{Self, Coin};
```

Comment:
Sui -> sui should probably be another PR. Is there one up for that?

Reply:
It is in progress, I decided to do this small bit and then proceed with other renamings.